data/json/vehicles/carts.json

```diff
@@ -255,6 +279,31 @@
     ],
     "items": [ { "x": 0, "y": 0, "chance": 30, "items": [ "goggles_welding" ] } ]
   },
+  {
+    "id": "motordriven_welding_cart",
+    "type": "vehicle",
+    "name": "Motor Driven Welding Cart",
+    "blueprint": [ "&" ],
+    "parts": [
+      {
+        "x": 0,
+        "y": 0,
+        "part": [
+          "frame_cover",
+          { "part": "tank_small", "fuel": "gasoline" },
+          "battery_motorbike",
+          "engine_vtwin",
+          "generator_7500w",
+          "wheel_caster",
+          "controls"
+        ]
+      }
+    ],
+    "items": [
```

Comment:
There's no storage in your welding cart.

Reply:
Shouldn't have any.